.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
```

Comment:
Not saying that it's wrong to have this file in the repo, but it probably should be in a separate PR.